src/etc/inc/certs.inc

```diff
@@ -2517,7 +2517,9 @@ function cert_build_list($type = 'cert', $consumer = '', $selectsource = false,
 	/* Create a hashed array with the certificate refid as the key and
 	 * descriptive name as the value. Exclude incompatible certificates. */
 	foreach ($config[$type] as $cert) {
-		if (cert_check_pkey_compatibility($cert['prv'], $consumer)) {
+		if (empty($cert['prv']) && $type == 'cert') {
```

Comment:
`$type == 'cert'` should be surrounded by parenthesis.

Reply:
done